wiki/Gameplay/Multiplayer/en.md

```diff
@@ -1,434 +1,380 @@
-If the teammate does **not** use that game modifier and made a small mistake enough to trigger the mod effect, the beatmap will continue as normal, and players using the mod will have their health bar interface drained and stuck to empty with no ill effect (meaning, visual effect only).
-If the player *using* the mod made a mistake and triggered it, the actual effect will occur and failed state will be triggered immediately.
+All players share the same health bar. If it depletes completely, the match ends. In `Tag Team Vs`, each team has their own health bars, and losing all health will cause that team to lose.
 
-When playing in windowed mode, do note that the cursor will **not** be bounded to the osu! window during the break/not the player's turn yet.
+Players using the [Sudden Death (SD)](/wiki/Game_modifier/Sudden_Death) or [Perfect (PF)](/wiki/Game_modifier/Perfect) game modifier will cause a fail for the whole team if they fail due to it.
 
-#### Result
+##### Result
 
-![](img/Multi_co-op_grade.jpg "Team Co-Op grade screen")
+![](img/Multi_co-op_grade.jpg "Team Coop results screen")
 
-![](img/Multi_tag_team_grade.jpg "Tag Team Vs grade screen")
+![](img/Multi_tag_team_grade.jpg "Tag Team Vs results screen")
 
-As stated before, score will not be counted for ranked score and performance points calculation because it is an *unranked* play.
+No ranked score or performance points are awarded for `Tag Coop` or `Tag Team Vs` matches.
 
-For Tag Coop, the result shown is the total score, hit score and accuracy of the team.
-Individual results will not be shown, and can only be seen in Match History.
-The result screen only appear if the beatmap was cleared successfully.
+For `Tag Coop`, the result shown is the total score, the [judgements](/wiki/Gameplay/Judgement) and accuracy of the team. Individual results are not shown, and can only be seen in the match history. The result screen only appears if the beatmap was cleared successfully, unlike other multiplayer modes.
 
-For Tag Team Vs result screen, refer to Team Vs's result screen explanation.
+For `Tag Team Vs` results screen, refer to `Tag Team Vs`'s [result screen explanation](#team-vs).
 
 ## Trivia
 
 ### History
 
-![](img/Multi_Mania_unpatched.jpg "Screenshot of Lobby of an unpatched osu! when osu!mania was released to public \(08 October 2012, 2012-10-08\)")
+::: Infobox
+![](img/Multi_Mania_unpatched.jpg "Screenshot of the main lobby when osu!mania was first released \(October 8, 2012\), but not yet playable in multiplayer")
+:::
 
-- Match Setup used to only allows up to a total of 8 players only.
-- osu!mania game mode was noted as `3` in Lobby for players that have not yet updated their osu! to the latest release patch (which enables osu!mania mode with no ranking support yet).
-- No Video was the *only* game modifier allowed for the players to use; however, it was removed in favour of its placement in the Visual Settings.
```

Comment:
does this imply that this setting only existed in the visual settings panel before also being added to the regular settings?

Reply:
https://github.com/ppy/osu-wiki/issues/7165